### PR TITLE
doc: wc add flow inputs

### PR DESCRIPTION
**Pull request by ruvenzx**

### CODE REVIEW

File: packages/web-component/README.md [hunk 1]
```diff
@@ -65,9 +67,9 @@ DESCOPE_BASE_URL
 DESCOPE_LOCALE=<locale>
 ```
 
-1. Run the sample `pnpm run start` / `pnpm run start-web-sample`
+1. Run the sample `pnpm run start`
```
Comment (asafshen) [inline packages/web-component/README.md]: why did you remove this?

File: packages/web-component/README.md [hunk 2]
```diff
@@ -41,9 +41,11 @@ render(){
 - Now you can add the custom element to your HTML
 
 ```html
-<descope-wc project-id="<project-id>" flow-id="<flow-id>"></descope-wc>
+<descope-wc project-id="<project-id>" flow-id="<flow-id>" form='{ "email": "predefinedname@domain.com", "myCustomInput": "12" }' client="+JSON.stringify({ browserName: window.navigator.appName, browserVersion: window.navigator.appVersion })+"></descope-wc>
```
Comment (asafshen) [inline packages/web-component/README.md:44]: you have an extrra `+` there which i think is redundant

also why do you use string notation in `form`  and json notation in `client`?


FYI I would give a smipler examples for `client

**Review by asafshen — COMMENTED** [review]
looks good! minor comment about the `+`

File: packages/web-component/README.md (hunk 2 above)
Comment (ruvenzx) [reply packages/web-component/README.md:44]: 1. fixed
2. I just copied this from the guide, can fix accordingly 
3. I will use only one key instead of 2

**Review by asafshen — APPROVED** [review]
great work!
